bin/mocha-test

```diff
@@ -2,6 +2,7 @@
 
 set -e
 
+export NODE_ENV=testing
```

Comment:
Will this affect the shell outside of this script?

What do you think about declaring this inline in the `mocha` call?

```bash
$ NODE_ENV=test ./node_modules/.bin/mocha ${tests_that_are_not_features}
```

Reply:
This is admittedly a Rails-ism, but what do you think about naming the test environment `test`?